common/cmake/devices.cmake

```diff
@@ -1327,13 +1338,17 @@ function(ADD_FPGA_TARGET)
     # Generate FASM
     # -------------------------------------------------------------------------
     set(OUT_FASM ${OUT_LOCAL}/${TOP}.fasm)
+    set(OUT_FASM_CONCATENATED ${OUT_LOCAL}/${TOP}.concat.fasm)
```

Comment:
`fasm2bels` keys off of `OUT_FASM`.  Make the `OUT_FASM` variable the final fasm file, etc.

Reply:
I am eventually copying the `top.concat.fasm` to `top.fasm` like (here)[https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1158/files#diff-2d565728ea2fbe9d3a017347718a0ba5R1349], so `OUT_FASM` is final.

Reply:
Sure.  Please copy the original un-cat'd file to a new name then.

Reply:
Ok, something like [this](https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1158/commits/3e9ef98df26c301fe2f0fc3adfccae8586b15bb0#diff-2d565728ea2fbe9d3a017347718a0ba5R1350)?

Reply:
Yep, perfect